examples/tutorials/data/hawc.py

```diff
@@ -0,0 +1,268 @@
+# For this reason, a single HAWC observation has several HDU index tables
+# associated to it, one per event type. In each table, there will be
+# paths to a distinct event list file and IRFs.
+# In the HAWC event data release, all the HDU tables are saved into
+# the same FITS file, and can be accesses through the choice of the hdu index.
+
+
+######################################################################
+# Load the tables
+
+data_path = "$GAMMAPY_DATA/hawc/crab_events_pass4/"
+hdu_filename = f"hdu-index-table-{energy_estimator}-Crab.fits.gz"
+obs_filename = f"obs-index-table-{energy_estimator}-Crab.fits.gz"
+
+# there is only one observation table
+obs_table = ObservationTable.read(data_path + obs_filename)
+
+# we read the hdu index table of fHit bin number 6
+fHit = 6
+hdu_table = HDUIndexTable.read(data_path + hdu_filename, hdu=fHit)
+
+
+######################################################################
+# From the tables, we can create a Datastore
+
+data_store = DataStore(hdu_table=hdu_table, obs_table=obs_table)
+
+data_store.info()
+
+
+######################################################################
+# There is only one observation
+
+obs = data_store.get_observations()[0]
+
+######################################################################
+# Select and peek events
+
+obs.events.peek()
+
+
+######################################################################
+# Peek the energy dispersion
+
+obs.edisp.peek()
+
+######################################################################
+# Peek the psf
+obs.psf.peek()
+
+######################################################################
+# Peek the background for one transit
+plt.figure()
+obs.bkg.reduce_over_axes().plot(add_cbar=True)
+
+######################################################################
+# Peek the effective exposure for one transit
+plt.figure()
+obs.aeff.reduce_over_axes().plot(add_cbar=True)
+
+
+######################################################################
+# Data reduction into a MapDataset
+# --------------------------------
+#
+# We will now produce a MapDataset using the data from one of the
+# fHit bins. In order to use all bins, one just needs to repeat this
+# process inside of a for loop that modifies the variable fHit.
+
+
+######################################################################
+# First we define the geometry and axes
+
+# create the energy reco axis
+# Note that this axis is the one used to create the background model map. If
+# different edges are used, the MapDatasetMaker will interpolate between
+# them, which might lead to unexpected behavior.
+energy_axis = MapAxis.from_edges(
+    [1.00, 1.78, 3.16, 5.62, 10.0, 17.8, 31.6, 56.2, 100, 177, 316] * u.TeV,
+    name="energy",
+    interp="log",
+)
+
+
+# and energy true axis
+energy_axis_true = MapAxis.from_energy_bounds(
+    1e-3, 1e4, nbin=140, unit="TeV", name="energy_true"
```

Comment:
is the edisp so poor at low energies that you need to start from 1e-3 TeV for 1TeV ereco?

Reply:
Ah indeed, this is just the same axis I always use (which matches the IRFs) - you're right that in principle one could start at e.g. 1e-2 TeV without much issue (see Fig 4 of https://arxiv.org/pdf/2109.04097.pdf). But the edisp is quite poor at low energies so I wouldn't go higher than that...